examples/rust/custom_space_view/src/color_coordinates_scene_parts.rs

```diff
@@ -34,7 +33,7 @@ impl ScenePartCollection<ColorCoordinatesSpaceView> for ColorCoordinatesScenePar
 /// Our scene(-parts) consist of single part which holds a list of egui-colors and their instance ids.
 #[derive(Default)]
 pub struct InstanceColors {
-    pub colors: Vec<(InstancePathHash, egui::Color32)>,
+    pub colors: Vec<(EntityPath, Vec<(InstanceKey, egui::Color32)>)>,
```

Comment:
This is a pretty complex type now, and it doesn't match the docstring of `InstanceColors`.

`use egui::Color32` could make it a bit shorter at least.